gtk/tests_backend/app.py

```diff
@@ -48,6 +48,16 @@ def content_size(self, window):
         content_allocation = window._impl.container.get_allocation()
         return (content_allocation.width, content_allocation.height)
 
+    def assert_current_window(self, window):
+        # Gtk 3.24.41 ships with Ubuntu 24.04 where present() works on Wayland
+        if self.IS_WAYLAND and self.GTK_VERSION < (3, 24, 41):
+            pytest.skip(
+                f"Assigning the current window is not supported for "
+                f"Gtk {'.'.join(map(str, self.GTK_VERSION))} on Wayland"
+            )
+        else:
+            assert self.app.current_window == window
```

Comment:
There is likely an older version of Gtk that works on Wayland to limit this with....but Gtk development is too opaque for me to try to track that down.

Reply:
I'm OK with this. There's so little consistency in Linux testing environments that I'm OK with this being a skip. If someone wants to suggest the limit can be lower, then can submit a PR.

However, given the actual assertion is a trivial comparison, I wonder if it might be better to handle this as a feature flag on the probe (i.e., a `supports_current_window_assignment`), rather than an abstracted assertion. Having 5 identical implementations of a trivial comparison, and one that is the same identical implementation with a skip check seems excessive. This would also allow us to continue the tests that *perform* a current-window check, but in the context of other useful behavior (i.e., we could make a decision on a per-test basis whether we want to skip the test, or ignore the check entirely.=)